src/assets/css/login.css

```diff
@@ -0,0 +1,164 @@
+form {
+  display: flex;
+  flex-direction: column;
+  margin-top: 48px;
+}
+
+img {
+  align-self: center;
+}
+
+input {
+  border-radius: 8px;
+  border: 1px black solid;
+  height: 40px;
+  margin-top: 4px;
+  padding: 0 8px;
+}
+
+.email, .password, .password-confirmation, .full-name {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 13px;
+}
+
+.container {
+  background: white;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  height: 425px;
+  padding: 47px 36px 0 36px;
+}
+
+.container-registration {
+  background: white;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  padding: 47px 36px 0 36px;
+}
+
+.background {
+  background-image: url('../../images/Black_Market_wallpaper.svg');
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+
+  align-items: flex-start;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  justify-content: flex-start;
+  padding: 66.5px 119px;
+}
+
+.login, .registration-page {
+  width: 360px;
+  gap: 20px;
+  display: flex;
+  flex-direction: column;
+}
+
+.login {
+  height: 575px;
+}
+
+.submit {
+  background: #00031A;
+  color: white;
+  height: 44px;
+}
+
+.submit:disabled {
+  background-color: #E0E0E0;
```

Comment:
wdyt about creating constanst for colors that you are gonna use later on, this is a good example